### ISSUE TITLE
Getting issue with Apigee module installation on Drupal 9.1.7

### ISSUE THREAD

**Issue by jyotierudkar** [apigee/apigee-devportal-kickstart-drupal#476]
On Installing modules Drupal server go down

**Comment by jyotierudkar** [comment]
On Installing modules Drupal server go down

**Comment by phdhiren** [comment]
@jyotierudkar 
Can you share error log around that?

You can checkout the [installation instruction from the documentation](https://www.drupal.org/docs/contributed-modules/apigee-developer-portal-kickstart/get-started-with-kickstart)

**Comment by NarendraShukla1989** [comment]
Please find below the log for the issue:

Error: Interface &#039;Apigee\Edge\Entity\EntityInterface&#039;  not found in include () (line 32 of /var/www/html/drupwl/modules/apigee_edge/src/Entity/EdgeEntityInterfsce.php).

@phdhiren

**Comment by phdhiren** [comment]
Please check above documentation to follow exact steps.

Above file does not exists in original distribution and make sure you use composer to install the module as given in documentation.

**Comment by jyotierudkar** [comment]
@phdhiren 
while following steps:

problems 1) composer installing apigee_edge module in a different location than drupal/module folder 2) the composer require drupa/apigee_edge runs fine but even after restarting drupal the module is not visible from Drupal UI>>extends>>module section

**Comment by phdhiren** [comment]
Can you share error log from Drupal log or php? The one shared earlier that file does not exists in original distribution. Can you share `composer show` to help in this more?

**Comment by jyotierudkar** [comment]
@phdhiren 
Module is getting installed in below location :
/root/.cache/composer/files/drupal

**Comment by phdhiren** [comment]
@jyotierudkar I hope you're using following the [documentation](https://www.drupal.org/docs/contributed-modules/apigee-developer-portal-kickstart/get-started-with-kickstart)

`composer create-project apigee/devportal-kickstart-project:9.x-dev MY_PROJECT --no-interaction`

The module should fall under web/modules/contrib directory if the `installer-paths` are correct
https://github.com/apigee/devportal-kickstart-project-composer/blob/77d66a5cde63b3b9d9ab69386416b4042320198c/composer.json#L65

**Comment by jyotierudkar** [comment]
@phdhiren we are trying to install apigee edge  module.. These all commands are for the apigee kickstart. 
Are those same for apigee edge

**Comment by phdhiren** [comment]
Yes, when you install apigee kickstart it enables you to evaluate the Apigee. From your comments it seems the dependencies are [not getting installed](https://github.com/apigee/apigee-devportal-kickstart-drupal/issues/476#issuecomment-842145434) or [not installed at correct place](https://github.com/apigee/apigee-devportal-kickstart-drupal/issues/476#issuecomment-848626286) for which composer is the best way. You should checkout apigee kickstart.

**Comment by jyotierudkar** [comment]
@phdhiren  checked in web/modules/contrib 
module is present there
but it is not reflecting in Drupal UI in Extend Tab


Also When we run command composer require drupal/apigee_edge
we are getting Nothing to install

**Comment by jyotierudkar** [comment]
@phdhiren 
we were able to finally get this sorted but while configuring the Apigee-edge, unable to configure it. It gives an error to update file and properties but does not let us proceed. 
We tried to follow this link to configure private file path  https://www.drupal.org/docs/contributed-modules/apigee-edge/configure-the-connection-to-apigee#configure-credentials-edge
Do we need to update any config or properties file before proceeding with Apigee-edge Module configuration?
